queue/queue.py

```diff
@@ -3,13 +3,14 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    return self.storage.insert(0, item)
   
   def dequeue(self):
-    pass
+    if len(self.storage):
+      return self.storage.pop()
 
   def len(self):
-    pass
+    return len(self.storage)
```

Comment:
nice job implementing the queue. to be completely correct I think you would want to utilize the size attribute. I'm sure your tests passed because it doesn't look like it was ever testing against self.size but it would be good practice to remember to add or subtract to the size depending on if you are enqueue or dequeue'ing.

Reply:
actually I just realized why your tests were passing. you should be returning self.size in the len function. if you did that your tests wouldn't pass because you hadn't incremented or decremented the self.size.